### DISCUSSION TITLE
Are the new version also on iOs?

### DISCUSSION THREAD

**Discussion by PauFCB** [leinelissen/jellyfin-audio-player#95] [Q&A] [answered]
Hi, I’m new on github, will the new iOs beta version with FLAC support be available in TestFlight or it’s the android version only? I have a large FLAC collecton in my Plex server and want to move to Jellyfin with this new beautiful app! Thanks

**Comment by leinelissen** [comment, marked as answer]
Hey @PauFCB! Yes, FLAC support is coming to TestFlight as well. The update is currently in review with Apple, who usually approve it within 48 hours.
